lib/ansible/module_utils/docker_common.py

```diff
@@ -83,16 +83,16 @@
 DEFAULT_TIMEOUT_SECONDS = 60
 
 DOCKER_COMMON_ARGS = dict(
-    docker_host=dict(type='str', aliases=['docker_url'], default=DEFAULT_DOCKER_HOST),
-    tls_hostname=dict(type='str', default=DEFAULT_TLS_HOSTNAME),
-    api_version=dict(type='str', aliases=['docker_api_version'], default='auto'),
-    timeout=dict(type='int', default=DEFAULT_TIMEOUT_SECONDS),
+    docker_host=dict(type='str', aliases=['docker_url'], default=DEFAULT_DOCKER_HOST, fallback=(env_fallback, ['DOCKER_HOST'])),
```

Comment:
Could you please update `lib/ansible/utils/module_docs_fragments/docker.py` to mention these variables, see `lib/ansible/utils/module_docs_fragments/vmware.py` to an example

Reply:
Yes, that would be a good idea. I guess we can get rid of the corresponding paragraph in the `notes:` section then.

@denraf Can you also update the documentation (in the docs fragment) for `cacert_path`, `cert_path` and `key_path`? They are a bit more tricky, since there the value of the env variable `DOCKER_CERT_PATH` is taken and modified.